test/functional/helpers.lua

```diff
@@ -23,28 +23,28 @@ local sleep = global_helpers.sleep
 local tbl_contains = global_helpers.tbl_contains
 local write_file = global_helpers.write_file
 
-local module = {
+local helper = {
   NIL = mpack.NIL,
   mkdir = lfs.mkdir,
 }
 
 local start_dir = lfs.currentdir()
-module.nvim_prog = (
+helper.nvim_prog = (
   os.getenv('NVIM_PRG')
   or global_helpers.test_build_dir .. '/bin/nvim'
 )
 -- Default settings for the test session.
-module.nvim_set = (
+helper.nvim_set = (
```

Comment:
Let's not rename the `module.` cases until we think of a better name than "helper".

Reply:
Do you want to leave `test/helper.lua` unchanged this time as well?

Reply:
IIRC I've used `global_helpers` for `tests/helper.lua` initially (when adding the pattern) - not a much better name, but the idea was the it helps with generated tags.
So here `functional_helpers` could be used.
But I agree that "helper" is very generic.